### ISSUE TITLE
Live server ISO boot error: "Failed to find installer media, failed to boot Clear Linux"

### ISSUE THREAD

**Issue by askervin** [clearlinux/distribution#1072]
How to reproduce:
```
$ qemu-system-x86_64 -enable-kvm -cpu host -smp 4 -m 1024 -cdrom clear-30500-live-server.iso -boot d
```
This prints finally "Failed to find installer media, retries exhausted, failed to boot Clear Linux". The same error is there at least up to version 30610, that is the latest at the time of reporting this issue.

There seems to be regression between versions 30490 and 30500, because 30490 finds the installer media without retries:
```
$ qemu-system-x86_64 -enable-kvm -cpu host -smp 4 -m 1024 -cdrom clear-30490-live-server.iso -boot d
```

My first impressions are that the `blkid -L CLR_ISO` fails to find the installer device in the init script. That might be due to `udev` files missing from the initrd. `blkid` seems to use `/dev/disk/by-label/CLR_ISO` node, which is probably created by udev. One difference between initrd images in 30490 and 30500 is that many udev related files have disappeared.

**Comment by mdhorn** [comment]
@askervin This is a duplicate of https://github.com/clearlinux/clr-installer/issues/484

The os-core bundle was changed and we lost the kmod binaries needed for detecting modular devices. We have updated the clr-installer, which generates the images, to now include the new os-core-plus bundle which brings kmod back into the initrd for the ISO images.

The fix will be in the next published images which should be 30630 or higher.

Just an FYI that if you want to ensure you have a validated ISO, only the ones which pass QA are published on the official downloads page: https://clearlinux.org/downloads
